pkg/controller/rest/verifiable/operation.go

```diff
@@ -42,16 +56,28 @@ func (o *Operation) GetRESTHandlers() []rest.Handler {
 func (o *Operation) registerHandler() {
 	o.handlers = []rest.Handler{
 		cmdutil.NewHTTPHandler(validateCredentialPath, http.MethodPost, o.ValidateCredential),
+		cmdutil.NewHTTPHandler(saveCredentialPath, http.MethodPost, o.ValidateCredential),
 	}
 }
 
-// ValidateCredential swagger:route POST /verifiable/validateCredential verifiable validateCredentialReq
+// ValidateCredential swagger:route POST /verifiable/credential/validate verifiable validateCredentialReq
```

Comment:
Not related to this PR, but we should chat about the base path.

Reply:
ok

Reply:
(how we best represent that these operations are the verifiable data registry).